aten/src/ATen/SparseTensorImpl.h

```diff
@@ -210,11 +210,8 @@ struct CAFFE2_API SparseTensorImpl : public TensorImpl {
     impl->refresh_numel();
     return impl;
   }
- private:
-  int64_t get_device_slow() const override {
-    return values_.get_device();
-  }
-
+private:
+    explicit SparseTensorImpl(at::TensorTypeId, const caffe2::TypeMeta&, at::Tensor indices, at::Tensor values);
```

Comment:
nit: no need for explicit

Reply:
the other constructor had it, but ya, I should have removed both.